config/charts/templates/kube-prometheus-stack.tpl

```diff
@@ -95,3 +110,19 @@ kubeStateMetrics:
     requests:
       cpu: 10m
       memory: 16Mi
+
```

Comment:
chart values: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L82

trying to replicate: https://github.com/w3f/base-services-charts/blob/master/charts/base-services/templates/rules/alertmanager.yaml#L33-L42